tbselenium/tbdriver.py

```diff
@@ -1,6 +1,7 @@
 import shutil
 from os import environ, chdir
 from os.path import isdir, isfile, join, abspath
+from os import name as currentOS
```

Comment:
this can be combined with line #3 (` from os import environ, chdir `)

Reply:
what you wanted to say is actually on line 2